### PR TITLE
Set up tutorial branch

### PR DESCRIPTION
**Pull request by chloewlin**
Since I translated the tutorial, I am going to finish nav as well. ;) Everything is consistent with `tutorial.md`.

### CODE REVIEW

File: content/tutorial/nav.yml [hunk 1]
```diff
@@ -1,111 +1,111 @@
...
           forceInternal: true
     - id: setup-for-the-tutorial
-      title: Setup for the Tutorial
+      title: 教學設定
       href: /tutorial/tutorial.html#setup-for-the-tutorial
       forceInternal: true
       subitems:
         - id: setup-option-1-write-code-in-the-browser
-          title: "Option 1: Write Code in the Browser"
+          title: 設定選項 1：在瀏覽器中寫程式碼
```
Comment (chloewlin) [inline content/tutorial/nav.yml]: Do we need quotation marks for option 1 and 2?

File: content/tutorial/nav.yml [hunk 2]
```diff
@@ -1,111 +1,111 @@
...
       forceInternal: true
       subitems:
         - id: lifting-state-up
-          title: Lifting State Up
+          title: 提升 State
           href: /tutorial/tutorial.html#lifting-state-up
           forceInternal: true
         - id: why-immutability-is-important
-          title: Why Immutability Is Important
+          title: Immutability（不可變性）的重要性
```
Comment (chloewlin) [inline content/tutorial/nav.yml]: We can either keep the English term or the Chinese or both. As long as it is consistent with `tutorial.md`.

File: content/tutorial/nav.yml (hunk 1 above)
Comment (neighborhood999) [reply content/tutorial/nav.yml]: It's fine for here, not need quotation marks.

File: content/tutorial/nav.yml (hunk 2 above)
Comment (neighborhood999) [reply content/tutorial/nav.yml]: Keep consistent is fine.

**Pull request merged**